gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/dichotomy/DichotomyRunner.java

```diff
@@ -58,12 +58,29 @@ public DichotomyRunner(FileImporter fileImporter, FileExporter fileExporter, Rao
         this.businessLogger = businessLogger;
     }
 
-    public DichotomyResult<RaoResponse> runDichotomy(CseValidRequest cseValidRequest, TTimestamp timestamp) {
+    public DichotomyResult<RaoResponse> runImportCornerDichotomy(CseValidRequest cseValidRequest, TTimestamp timestamp) {
         int npAugmented = timestamp.getMNII().getV().intValue();
         int np = timestamp.getMiBNII().getV().intValue() - timestamp.getANTCFinal().getV().intValue();
         double maxValue = (double) npAugmented - np;
         Network network = importNetworkFile(cseValidRequest);
-        String jsonCracUrl = getJsonCracUrl(cseValidRequest, network);
+        String jsonImportCracUrl = getJsonImportCracUrl(cseValidRequest, network);
+        businessLogger.info(DICHOTOMY_PARAMETERS_MSG, DEFAULT_MIN_INDEX, (int) maxValue, (int) DEFAULT_DICHOTOMY_PRECISION);
+        DichotomyEngine<RaoResponse> engine = new DichotomyEngine<>(
+                new Index<>(DEFAULT_MIN_INDEX, maxValue, DEFAULT_DICHOTOMY_PRECISION),
+                INDEX_STRATEGY_CONFIGURATION,
+                getNetworkShifter(timestamp.getSplittingFactors(), network, cseValidRequest),
+                getNetworkValidator(cseValidRequest, jsonImportCracUrl));
+        return engine.run(network);
+    }
+
+    public DichotomyResult<RaoResponse> runExportCornerDichotomy(CseValidRequest cseValidRequest, TTimestamp timestamp, boolean isExportCornerActive) {
```

Comment:
Could be interesting to see if we can mutualise "runImportCornerDichotomy" and "runExportCornerDichotomy" (maybe it is too difficult or makes it too difficult to read, but at least we can try ^^)